### PR TITLE
Release v7.1.0r

### PR DESCRIPTION
**Pull request by bekozi**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
-    skip: True  # [win or py3k]
+    number: 0
+    skip: True  # [win]
 
 requirements:
     build:
         - python
         - numpy
-        - esmf
+        - esmf ==7.1.0r
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We should do `esmf {{ version }}` here and below to avoid errors when updating the package.

Comment (bekozi) [reply recipe/meta.yaml]: 👍 Pushed an update. Thanks for the review!

**Comment by bekozi** [comment]
@ocefpaf Is there something going on with infrastructure? Am I missing something obvious with these recipes? :confounded:

**Comment by ocefpaf** [comment]
Let's rerender and find out :smile: 

@conda-forge-admin, please rerender

**Comment by bekozi** [comment]
SF is bad with tarballs/zips. Is it okay use use a git repo and tag?

Re-rendering seemed to help, so thank you!

**Comment by ocefpaf** [comment]
> SF is bad with tarballs/zips. Is it okay use use a git repo and tag?

That is fine for now. Do you have a plan to move away from `sourceforge`?

> Re-rendering seemed to help, so thank you!

We will also need to refresh the CircleCI token. But before we do that I'd like to point this feedstock to @CJ-Wright, @justcalamari, and @isuruf in case they want to take a look at CircleCI and check if we can automate the token refresh using the bot.

**Comment by bekozi** [comment]
> move away from sourceforge

:anguished: That's the plan but who knows if/when that will happen...

**Comment by isuruf** [comment]
Thanks @ocefpaf, "There are no deploy keys for this repository."

**Comment by ocefpaf** [comment]
> Thanks @ocefpaf, "There are no deploy keys for this repository."

@isuruf let me know if you want to experiment with this feedstock or if it is OK to issue the token update.

@bekozi sorry, we are using the fact that this one was not updated in a while to experiment with some new tech :smile:

**Comment by bekozi** [comment]
> @bekozi sorry, we are using the fact that this one was not updated in a while to experiment with some new tech smile

No problem. Take your time. Always good to see it from the perspective of a guinea pig once in awhile. :grin:

**Comment by CJ-Wright** [comment]
@isuruf is that from the circle log? Is there a good way to access the log via program?

**Comment by isuruf** [comment]
No, that's from the settings of this repo. You can use the github API at https://developer.github.com/v3/repos/keys/

**Comment by CJ-Wright** [comment]
Is the division between things that don't have keys, and those that do or is there a third state of having keys that are out of date?

**Comment by ocefpaf** [comment]
> Is the division between things that don't have keys, and those that do or is there a third state of having keys that are out of date?

FYI I just updated the token here. People needed this version to the uploaded.

**Pull request merged**